### DISCUSSION TITLE
Qual a melhor library para capturar um fingerprint do browser?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#576] [Perguntas e Respostas]
A mais facilmente encontrável na Web é o [fingerprintjs2](https://www.npmjs.com/package/fingerprintjs2), vocês conhecem outras alternativas?

**Comment by kivervinicius** [comment]
@felquis eu uso o fingerprintjs2, acredito ser uma boa solução, pois utiliza também canvas fingerprint. até hoje em meus clientes não se repetiram nenhuma chave, e a grande maioria não muda, hoje uso como um uuid para analytics somente.
